integration-cli/docker_cli_swarm_test.go

```diff
@@ -1316,3 +1316,18 @@ Resources:
 	c.Assert(err, checker.IsNil, check.Commentf(out))
 	c.Assert(out, checker.Contains, expectedOutput, check.Commentf(out))
 }
+
+func (s *DockerSwarmSuite) TestSwarmUpdateWithoutArgs(c *check.C) {
```

Comment:
There will be soon enough :angel: (see #29143)
@yongtang can you add a comment on that one that says "TODO migrate to a unit test". As soon as #29143 is in, I'll do the `swarm` command